onnxruntime/core/providers/dml/DmlExecutionProvider/src/GraphKernelHelper.cpp

```diff
@@ -105,11 +105,11 @@ namespace GraphKernelHelper
     bool GetGraphInputConstness(
         uint32_t index,
         const onnxruntime::OpKernelInfo& kernelInfo,
-        const onnxruntime::ConstPointerContainer<std::vector<onnxruntime::NodeArg*>>& fusedNodeInputDefs,
```

Comment:
>ConstPointerContainer [](http://example.com/codeflow?start=27&length=21)

This `ConstPointerContainer` looks beneficial to keep if we're using vector, but even better yet is just `gsl::span<const std::string>`.

Reply:
Actually yes, gsl::span makes more sense.